### ISSUE TITLE
Excellent for Samsung Windfree ACs

### ISSUE THREAD

**Issue by fate8383** [teodoradima/homebridge-samsung-air-conditioner-smart-things#7]
I have been looking for this many months, finally something works with this air condition I have. I have been using smartthings plugin and hated it every day and night as it is completely unreliable, does not sync status and also the plugin stopped being maintained. This is excellent and after I confirmed/tested it is working, I even deleted completely smartthings plugin from homebridge. Thanks a lot!

**Comment by gvchagas** [comment]
I agree with everything that has been said!  it's a great plugin and I was finally able to use the air conditioning in HomeKit.  as a suggestion, it would be interesting for the air conditioner's internal thermostat to be shown as a separate device, since as it is it is not possible to create automations in HomeKit based on temperature.

**Comment by paqpaqpaq** [comment]
> I agree with everything that has been said! it's a great plugin and I was finally able to use the air conditioning in HomeKit. as a suggestion, it would be interesting for the air conditioner's internal thermostat to be shown as a separate device, since as it is it is not possible to create automations in HomeKit based on temperature.

https://github.com/0x4a616e/homebridge-smartthings-ac#readme
That one does so

**Comment by fate8383** [comment]
> > I agree with everything that has been said! it's a great plugin and I was finally able to use the air conditioning in HomeKit. as a suggestion, it would be interesting for the air conditioner's internal thermostat to be shown as a separate device, since as it is it is not possible to create automations in HomeKit based on temperature.
> 
> https://github.com/0x4a616e/homebridge-smartthings-ac#readme That one does so

this one does not work with wind free model, at least for me.

**Comment by paqpaqpaq** [comment]
@fate8383 that is weird, because it does so here.

{
            "token": "xxxxxxxxxx",
            "minTemperature": 16,
            "maxTemperature": 30,
            "updateInterval": 15,
            "platform": "HomebridgeSmartThingsAC"
        },

**Comment by fate8383** [comment]
> @fate8383 that is weird, because it does so here.
> 
> { "token": "xxxxxxxxxx", "minTemperature": 16, "maxTemperature": 30, "updateInterval": 15, "platform": "HomebridgeSmartThingsAC" },

https://github.com/0x4a616e/homebridge-smartthings-ac/issues/3#issue-959076144 I have this issue and will not install another hap-nodejs just to not mess things up

**Comment by paqpaqpaq** [comment]
Just install the lib I would say :)

The issue with this (topic) plugin, is that it only fetches the state when restarting HB.
In other words: if you switch on the AC, which it will, and close the home app, re-open again, the home app will say it is off. 
This could be problematic when trying to make changes when away. 
Also if you would change something when running (and home claims it is off) it needs to resend all commands agan. So if you even open temp setting or mode it is in, it starts sending commands. 
Just makes no sense.

**Comment by fate8383** [comment]
> Just install the lib I would say :)
> 
> The issue with this (topic) plugin, is that it only fetches the state when restarting HB. In other words: if you switch on the AC, which it will, and close the home app, re-open again, the home app will say it is off. This could be problematic when trying to make changes when away. Also if you would change something when running (and home claims it is off) it needs to resend all commands agan. So if you even open temp setting or mode it is in, it starts sending commands. Just makes no sense. #4
> 
> That is why I switched to the other plugin, which behaves as one would expect.

ok took the advice and switched to the other plugin by installing the lib, it works. thanks
